modules/codegen-cli/src/smithy4s/codegen/cli/CodegenCommand.scala

```diff
@@ -125,7 +125,8 @@ object CodegenCommand {
             excludedNS,
             repositories.getOrElse(List.empty),
             dependencies.getOrElse(List.empty),
-            transformers.getOrElse(List.empty)
+            transformers.getOrElse(List.empty),
+            localJars = List.empty
```

Comment:
Could we actually make that a CLI argument, for completeness. You can follow the exact same patterns as the other ones that have `getOrElse(List.empty)` above this one